app/views/foreman_ansible/job_templates/ansible_roles_-_install_from_git.erb

```diff
@@ -23,6 +23,9 @@ model: JobTemplate
 ---
 - hosts: all
   tasks:
-    - command: git clone <%= input('git_repository') %> <%= input('location') %>
+    - git:
+        repo: <%= input('git_repository') %>
```

Comment:
Would it be safer to add quotes? just as additional safety measure, unrelated to the actual change.
```suggestion
        repo: "<%= input('git_repository') %>"
```

Reply:
@ezr-ondrej agreed. Modified accordingly.